src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int COLORS_COUNT = ColorList.values().length;
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(COLORS_COUNT);
```

Comment:
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked